kotlin/wallet-core-kotlin/src/jsMain/kotlin/com/trustwallet/core/JsAnySigner.kt

```diff
@@ -0,0 +1,14 @@
+package com.trustwallet.core
```

Comment:
can you add license header for new files?

Reply:
Done ✅